tests/test_advisory_opinions.py

```diff
@@ -35,13 +35,14 @@ def test_parse_ao_citations(text, ao_nos, expected):
 
 
 @pytest.mark.parametrize("text,expected", [
-    ("2 U.S.C. 432h", set([(52, 30102)])),
-    ("52 U.S.C. 30116a", set([(52, 30116)])),
-    ("2 U.S.C. 441b, 441c, 441e", set([(2, 441)])),
-    (" 2 U.S.C. §437f", set([(52, 30105)])),
-    ("52 U.S.C. § 30101", set([(52, 30101)])),
-    (" 2 USC §437f **test with no . in USC**", set([(52, 30105)])),
-    ("52 U.S.C. § 30101 **newline needed to ensure two entries**,\n 52 USC. § 30101 other words", set([(52, 30101)])),
+    ("2 U.S.C. 432", set([(52, '30102')])),
+    ("52 U.S.C. 30116(a", set([(52, '30116')])),
```

Comment:
Now that we allow alphabets and hyphens in the section, we should add a test like:
```
("52 U.S.C. 30116a", set([(52, '30116a')])),
```

Reply:
Thank you! I had added a test for 2 U.S.C. §441a-1 == 52 U.S.C. §30117 but I somehow lost it when re-working the (very stale) branch. I'll add it back.

Reply:
Oh nevermind, I see it now. I'll add a "52 U.S.C. 30116a === 52 U.S.C. 30116a" test too.